array-map/main.js

```diff
@@ -0,0 +1,19 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+const prices = numbers.map(x => '$' + x + '.00');
```

Comment:
While this works for this situation it would not come out correctly if any of the numbers had decimals already. A better solution would be to use `x => '$' + x.toFixed(2)`. That will work with any number.